examples/lighting-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -147,3 +148,13 @@ void emberAfOnOffClusterInitCallback(EndpointId endpoint)
     ESP_LOGI(TAG, "emberAfOnOffClusterInitCallback");
     GetAppTask().UpdateClusterState();
 }
+
+void AppDeviceCallbacksDelegate::OnIPv4ConnectivityEstablished()
```

Comment:
does the M5 stack only support IPV4?

Reply:
When testing it seems it supports both. This was taken from what all-clusters-app does.